datadog_checks_dev/datadog_checks/dev/tooling/templates/integration/check/{check_name}/manifest.json

```diff
@@ -2,23 +2,24 @@
   "manifest_version": "2.0.0",
   "app_uuid": "{app_uuid}",
   "app_id": "{integration_id}",
-  "display_on_public_website": false,
+  "display_on_public_website": {display_on_public_website},
   "tile": {{
     "overview": "README.md#Overview",
     "configuration": "README.md#Setup",
     "support": "README.md#Support",
     "changelog": "CHANGELOG.md",
-    "description": "",
+    "description": "<FILL IN>",
     "title": "{integration_name}",
-    "media": [],
+    "media": {media},
```

Comment:
Does this mean it will link to the outline that's included in the assets/dashboard manifest? ex:
    "media": [
      {
        "media_type": "image",
        "caption": "FILL IN Image 1 caption",
        "image_url": "<FILL IN>"
      },
      {
        "media_type": "image",
        "caption": "FILL IN Image 2 caption",
        "image_url": "<FILL IN>"
      },
      {
        "media_type": "image",
        "caption": "FILL IN Image 3 caption",
        "image_url": "<FILL IN>"
      }

Reply:
This means that we will fill that field differently for integrations that partners create vs those that get created in ints-core.

What you posted will show up for partners, whereas for ints-core we'll still have the empty list.

Reply:
Thank you!